src/js/components/bulkDownload/accounts/AccountDataContent.jsx

```diff
@@ -110,6 +112,7 @@ export default class AccountDataContent extends React.Component {
                             currentAccountLevel={accounts.accountLevel}
                             updateFilter={this.props.updateFilter}
                             valid={accounts.accountLevel !== ''} />
+                        <DefCodeFilter type="accounts" />
```

Comment:
We need to update the user selections at the bottom of the form. `Your selected options are...`